autoload/vebugger.vim

```diff
@@ -268,26 +270,39 @@ function! vebugger#createDebugger(command)
     return l:debugger
 endfunction
 
+if has('timers')
+    function! ReadingTimerCallback(debugger, timer_id)
```

Comment:
Ah OK, I thought that `function` in an expression would always create a partial - in your example a partial that does not bind any argument. In my tests, I used `start_timer` with a string of the function name directly:
```
timerId = start_timer(500, 's:foo')
```
That syntax only works with global functions. Sorry, I'm not very familiar with vimscript but I'm getting better I think. :) 

Since your example works in neovim, everything is fine and I will update my patch tomorrow.